Frameworks/Foundation/NSKVOSwizzling.mm

```diff
@@ -385,6 +406,10 @@ static void _NSKVOEnsureCollectionWillNotify(id object, NSString* key, const cha
                                formatSelector(@"replace%sAtIndexes:with%s:", rawKey, rawKey),
                                key,
                                (IMP)NSKVONotifying$replaceXxxAtIndexes$withXxx$);
+    replaceAndAssociateWithKey(object, formatSelector(@"add%s:", rawKey), key, (IMP)NSKVONotifying$addXxx$);
```

Comment:
is swizzling every method appropriate here?

we're only supposed to swizzle in the array case if it has an insertion method. this changes that logic.